chia/wallet/puzzles/augmented_condition.clvm

```diff
@@ -0,0 +1,26 @@
+(mod
+  (
+    CONDITION
+    INNER_PUZZLE_HASH
```

Comment:
This is the puzzle that is used by the taker to claim the coin after the timelock has passed. So when we're creating the puzzles for the clawback coin we don't have access to the taker's full puzzle - we'll just be given an address which we convert to puzhash and curry in.

Reply:
You can still create this puzzle hash even if you don't know the recipient's puzzle reveal:
```
AUGMENTED_CONDITION_MOD.curry(condition, recipient_puzzle_hash).get_tree_hash_precalc(recipient_puzzle_hash)
```

Reply:
hey I didn't know about precalc, thanks. I think that also means we can skip providing the inner puzzle in the solution since we'll prove we know it via the puzzle reveal. Have modified accordingly